CRM/Contact/BAO/Query.php

```diff
@@ -4336,6 +4336,8 @@ public static function getQuery($params = NULL, $returnProperties = NULL, $count
    * @param string $apiEntity
    *   The api entity being called.
    *   This sort-of duplicates $mode in a confusing way. Probably not by design.
+   * @param bool $useGroupBy
+   *   Should the query be grouped by contact id?
```

Comment:
How about instead of a question you use "determines whether the result will be grouped by contact ID" if that makes sense. Questions in parameter descriptions sound a bit like it doesn't know what it does :smile: